pychromecast/controllers/media.py

```diff
@@ -297,10 +300,16 @@ def update_status(self, callback_function_param=False):
         self.send_message({MESSAGE_TYPE: TYPE_GET_STATUS},
                           callback_function=callback_function_param)
 
-    def _send_command(self, command):
+    def _send_command(self, command, blocking=True, timeout=None):
```

Comment:
I don't like this to be added to every command, instead it is up to the developer using the lib to call a wait method.